operations/node_management/S3FS_Usage_and_Guidelines.md

```diff
@@ -0,0 +1,64 @@
+# S3FS Usage and Guidelines for Shasta
+
+## Introduction
+
+S3FS is being deployed as tool to provide temporary relief of space usage as well as supporting SDU/NMD services as a near-posix file system to provide a landing point for dumps.
```

Comment:
Change to:

S3FS is being deployed as tool to provide temporary relief of space usage, as well as supporting the System Diagnostic Utility (SDU) and Node Memory Dump (NMD) services as a near-posix file system to provide a landing point for dumps.